client/src/utils/dbAPI.js

```diff
@@ -13,6 +13,9 @@ export default {
   createCampaign: campaignData => {
     return axios.post("/api/campaign", campaignData);
   },
+  viewCampaign: id => {
```

Comment:
Not something in the api, an actual link on each campaign card that points to /viewcampaign/:id